sdk/identity/azure-identity/images/mermaidjs/DefaultAzureCredentialAuthFlow.md

```diff
@@ -5,24 +5,24 @@
 %% 2. Run command: mmdc -i DefaultAzureCredentialAuthFlow.md -o DefaultAzureCredentialAuthFlow.svg
 
 flowchart LR;
```

Comment:
Nit: Please install v8.14.0 of the Mermaid CLI and then regenerate the SVG. It fixes this issue: 
![image](https://user-images.githubusercontent.com/10702007/216447897-a4358a7d-f4b2-4523-ba5f-7f4c94a62de4.png)

And while you're in here, it probably makes sense to update the Markdown comment to say v8.14.0 was used to generate the SVG.